android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/components/AppDrawer.kt

```diff
@@ -216,7 +219,8 @@ private fun OtherPatientsItem(
     onSideMenuClick(
       AppMainEvent.OpenRegistersBottomSheet(
         registersList = navigationConfiguration.bottomSheetRegisters?.registers,
-        navController = navController
+        navController = navController,
+        title = navigationConfiguration.bottomSheetRegisters?.display!!
```

Comment:
This has a potential `NullPointerException` with the `!!` on the `display` property. Make `title` nullable instead. Then apply an if condition for when `title.isNullOrEmpty()` - use default `Other patients` string.